docs/02_cleos/02_how-to-guides/how-to-buy-ram.md

```diff
@@ -3,21 +3,26 @@
 Acquire RAM for contract deployment, database tables, and other blockchain resources
 
 ## Before you begin
+Before you follow the step, make sure you fulfill the following items: 
 
-* You have an account
+* You have an EOSIO account
 
-* Ensure the reference system contracts from `eosio.contracts` repository is deployed and used to manage system resources
+* Ensure the reference system contracts from [`eosio.contracts`](https://github.com/EOSIO/eosio.contracts) repository is deployed and used to manage system resources
 
-* You have sufficient token allocated to your account
+* You have sufficient [token allocated](how-to-transfer-an-eosio.token-token.md) to your account
 
 * Install the currently supported version of `cleos`
 
-* Unlock your wallet
+[[info | Note]]
+| The cleos tool is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the cleos tool. 
+
+* Unlock your wallet using the `cleos wallet unlock` command
+
 
 ## Steps
 
```

Comment:
We should have a command section like `cleos create account` describe the meaning of each parameter.

Reply:
For the `cleos create account` command, we have explained the parameters in this PR:

https://github.com/EOSIO/eos/pull/9933/files 

Does that address your suggestion here? @huangminghuang